src/primitives.jl

```diff
@@ -206,22 +206,6 @@ unwrapped `Nullable` entries.
 dropnull!(X::NullableVector) = deleteat!(X, find(X.isnull)).values # -> Vector
 
 """
-    anynull(X)
-
-Returns whether or not any entries of `X` are null.
-"""
-anynull(X::Any) = any(_isnull, X)           # -> Bool
-anynull(X::NullableArray) = any(X.isnull)   # -> Bool
```

Comment:
It would affect all types, but only when `isnull` is constant for the input type. So `any(X.isnull)` is still faster than the fallback for `NullableArray`, since it avoids the creation of a `Nullable` by `getindex`.